src/gui/BandSplitter/BandSplitterPlot.cpp

```diff
@@ -111,6 +111,15 @@ BandSplitterPlot::BandSplitterPlot (State& pluginState,
     for (int bandIndex = 0; bandIndex < numBands; ++bandIndex)
         setFilterActive (bandIndex, true);
 
+    getCutoffParam = [this](int bandIndex) -> const chowdsp::FreqHzParameter::Ptr& {
+        if (bandIndex < (numBands / 3))
+            return bandSplitterParams.cutoff;
+        else if (bandIndex <= numBands / 2)
+            return bandSplitterParams.cutoff2;
+        else
+            return bandSplitterParams.cutoff3;
+    };
```

Comment:
We should probably keep the comment that was with this logic previously, just to help avoid confusion in the future.